src/oas/transformers/schema/keywords/format.ts

```diff
@@ -15,6 +15,16 @@ const ranges = {
 
 const createNumberFormatter = (min: number, max: number): Converter => {
   return schema => {
+    if (schema.minimum === undefined) {
```

Comment:
nit: these can be simplified to a straight assignment

Reply:
🤦 you are right - fixed